- Add workaround python 3.6 bug with datetime.fromtimestamp(0) (https://bugs.python.org/issue29097)
